global-script.lua

```diff
@@ -1664,14 +1664,14 @@ end
 function BoardSetup()
     if getMapCount({norm = true, them = true}) == 0 then
         if isThematic() then
-            MapPlacen(posMap[numBoards][boardLayout],rotMap[numBoards][boardLayout])
+            MapPlacen(boardLayouts[numBoards][boardLayout])
         else
             StandardMapBag.shuffle()
             if scenarioCard ~= nil and scenarioCard.getVar("boardSetup") then
-                local tables = scenarioCard.call("BoardSetup", { boards = numBoards })
-                MapPlacen(tables.posTable,tables.rotTable)
+                local layout = scenarioCard.call("BoardSetup", { boards = numBoards })
```

Comment:
instead of storing this in a local variable, you can just pass the value straight into the function